src/DonationForms/ViewModels/DonationFormViewModel.php

```diff
@@ -108,6 +109,11 @@ public function enqueueGlobalStyles()
             }"
         );
 
+        wp_add_inline_style(
+            'givewp-base-form-styles',
+            give_get_option('custom_form_styles', '')
```

Comment:
Just want to make sure, we don't need to sanitize this do we? I think `wp_add_inline_style` handles that but want to make sure.

Reply:
From what I can see, `wp_add_inline_style` outputs whatever you pass to it directly. Applied an `wp_strip_all_tags` surrounding the option value.